src/lib/zapUtils.ts

```
@@ -0,0 +1,78 @@
+export function useFetchImplementation(fetchImplementation) {
+  _fetch = fetchImplementation;
+}
+
+export async function getZapEndpoint(metadata) {
+  try {
+    let lnurl = '';
+    let { lud06, lud16 } = JSON.parse(metadata.content || '{}'); // Add fallback for missing content
+    console.log("Parsed metadata:", { lud06, lud16 });
+
+    if (lud06) {
+      let { words } = bech32.decode(lud06, 1000);
```

Biome

[error] This let declares a variable that is only assigned once.

'words' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)